ocaml-lsp-server/src/custom_requests/req_merlin_jump.mli

```diff
@@ -0,0 +1,21 @@
+open Import
+
+module Request_params : sig
+  type t
+
+  val yojson_of_t : t -> Json.t
+
+  val create
+    :  text_document:TextDocumentIdentifier.t
+    -> position:Position.t
+    -> target:string
+    -> unit
+    -> t
+end
+
+type t
+
+val t_of_yojson : Json.t -> t
```

Comment:
Is this a requirement for the custom request ? it doesn't look like its in use.

Reply:
Yes this is right. I'll get rid of it.